openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -78,7 +78,7 @@ def createODB(*argv):
 		dofList = [1, 2, 3]
 	
 	# Save node and element data in the main Output folder
-	idbf.saveNodesandElements(ODBdir)
+	idbf.saveNodesandElements(ModelName, LoadCaseName)
```

Comment:
This input argument should be ODBdir since nodes and element data does not change with a load case. It should be saved in the main ODB folder and not in the load case-specific folder.

Reply:
Sounds good. This will require that I modify saveNodesandElements to save in the ODBdir. I'll make the change and update.